backend/src/main/java/de/neuefische/ffmjava221/teamprojekt/backend/employee/NewEmployee.java

```diff
@@ -0,0 +1,9 @@
+package de.neuefische.ffmjava221.teamprojekt.backend.employee;
+
+public record NewEmployee(
+        String name
+) {
+    public Employee withID(String id) {
```

Comment:
Besser wäre:
```suggestion
    public Employee withId(String id) {
```
(Kamelschreibweise)

Reply:
fixed